pages/common/trash-cli.md

```diff
@@ -0,0 +1,23 @@
+# trash-cli
+
+> Trash-cli is a less permanent way of removing files via the command line.
```

Comment:
"less permanent" seems a bit unclear. Perhaps - "A command line interface to the trashcan APIs" sounds better.

Reply:
Sounds good.